controllers/userController.js

```diff
@@ -0,0 +1,68 @@
+const userService = require('../services/userService');
+
+const userEmailCheck = async (req, res) => {
+  try {
+    const { email } = req.body;
+    if (!email) return res.status(400).send('EMAIL_EMPTY!');
+
+    const result = await userService.userEmailCheck(email);
+    if (!result) return res.status(400).json({ isEmailExist: false });
+
+    return res.status(200).json({ isEmailExist: true });
+  } catch (err) {
+    console.log(err);
+    err = new Error('CONTROLLER_ERROR');
+    err.statusCode = 400;
+    throw err;
+  }
+};
+
+const login = async (req, res) => {
+  try {
+    const { email, password } = req.body;
+    if (!email || !password) {
+      return res.status(400).send(false);
+    }
+    const result = await userService.login(email, password);
+    if (!result) return res.status(400).json({ passwordError: 'CHECK_PASSWORD' });
+    return res.status(200).send({ accessToken: result });
+  } catch (err) {
+    console.log(err);
+    err = new Error('CONTROLLER_ERROR');
+    err.statusCode = 400;
```

Comment:
```suggestion
  } catch (err) {
    err = new Error('INVALID_USER_INPUT');
    err.statusCode = 400;
```

- 위와 같이 컨트롤러단에서 나는 에러의 포괄적인 원인을 에러메세지에 담아도 좋겠네요!